### ISSUE TITLE
Work with Refs

### ISSUE THREAD

**Issue by AngelLuisSanchez** [objectify/objectify#371]
Hi everyone!

I'm having some problems with Ref. This is my example code:

```
@Entity
public class Country{
    @Id Long id;
    @Index String name;
    String code;
}

@Entity
public class Signature{
    @Id long id;
    @Load Ref<Country> country;
}
```

How can I create a Signature object??
I've tried to load Country object by query filter, like this:
`Contry country= ofy().load().type(Contry .class).filter("name =", name).first().now();`
In this way, I obtain object, not REF to create Signature.

How can I reach this?

Thanks!

**Comment by stickfigure** [comment]
Hi! Can you rephrase the question? I'm afraid I don't understand what you are asking.

**Comment by UrbanChrisy** [comment]
```Java
Signature sig = new Signature();

Contry country = ofy().load().type(Contry .class).filter("name =", name).first().now();
Ref<Country> countryRef = Ref.create(country);
sig.setCountry(countryRef);

ofy().save().entity(sig);
```

Is this what you are meaning?
